cellpose/io.py

```diff
@@ -560,6 +561,21 @@ def masks_flows_to_seg(images, masks, flows, file_names, diams=30., channels=Non
 
     np.save(base + "_seg.npy", dat)
 
+def save_features_csv(file_name):
+    """ save features to .csv file and remove if it already exists
+    Args:
+        file_name (str): Target CSV file name
+
+    Returns:
+        None
+    """
+    file_name = os.path.splitext(file_name)[0] + "_cp_features.csv"
+    if os.path.exists(file_name):
+        os.remove(file_name)
```

Comment:
It might be nice to ask the user if they really want to replace the file before removing it